src/overview/bare-metal.md

```diff
@@ -1,59 +1,59 @@
-# Bare Metal (`no_std`)
+# Developing on Bare Metal (`no_std`)
 
-Using `no_std` may be more familiar to embedded Rust developers; it does not use `std` (the Rust standard library) but instead uses a subset, the `core` library. The [official Rust embedded book] has a [great section on this].
+Using `no_std` may be more familiar to embedded Rust developers; it does not use `std` (the Rust [`standard`][rust-lib-std] library) but instead uses a subset, the [`core`][rust-lib-core] library. [The Embedded Rust Book][embedded-rust-book] has a great [section][embedded-rust-book-no-std] on this.
 
-It's important to note that in general a `no_std` crate can always compile in `std` environment but the inverse is not true. Therefore, when creating crates it's worth keeping in mind if it needs the standard library to function.
+It's important to note that since `no_std`uses the Rust core library, a subset of the Rust standard library,  a `no_std` crate can compile in `std` environment but the oposite is not true. Therefore, when creating crates it's worth keeping in mind if it needs the standard library to function.
 
-[great section on this]: https://docs.rust-embedded.org/book/intro/no-std.html
-[official rust embedded book]: https://docs.rust-embedded.org/
 
-## Hardware Abstraction Layers
+[embedded-rust-book]: https://docs.rust-embedded.org/
+[embedded-rust-book-no-std]: https://docs.rust-embedded.org/book/intro/no-std.html
+[rust-lib-core]: https://doc.rust-lang.org/core/index.html
+[rust-lib-std]: https://doc.rust-lang.org/std/index.html
 
-Previously, the primary focus of `no_std` development was the ESP32 and (to a lesser extent) the ESP8266.
 
-Now there is a renewed effort to implement `no_std` support for the entire lineup of Espressif devices from the ESP32 and newer. These new HALs can be found in the [esp-rs/esp-hal] repository.
+## Current support
 
-There is also some level of support for Wi-Fi and Bluetooth via [esp-rs/esp-wifi] for ESP32, ESP32-C3, ESP32-S2, and ESP32-S3.
+The table below covers the current support for `no_std` at this moment for different Espressif products.
 
-[esp-rs/esp-hal]: https://github.com/esp-rs/esp-hal
-[esp-rs/esp-wifi]: https://github.com/esp-rs/esp-wifi
+> **Notes**:
+>
+> - ✅ - the feature is implemented or supported
+> - ⏳ - the feature is under development
+> - ❌ - the feature is not supported
+> - [ESP8266 HAL][esp-rs/esp8266-hal] is in maintenance mode and no further development will be done for this chip
 
-## Chip Support
+|         | [HAL][esp-rs/esp-hal] | [PAC][esp-rs/esp-pacs] | [Wi-Fi][esp-rs/esp-wifi] | [Bluetooth][esp-rs/esp-wifi] | [Heap][esp-rs/esp-alloc] | [Println][esp-rs/esp-println] | [Backtrace][esp-rs/esp-backtrace] | [Storage][esp-rs/esp-storage] |
+| -------- | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
+|  ESP32   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |
+| ESP32-C2 |   ✅   |   ✅   |   ✅   |   ✅   |   ❌   |   ✅   |   ✅   |   ✅   |
```

Comment:
I don't know why C2 and C6 are marked as not supported, but they are. `esp-alloc` doesn't even have features per-chip.